source/whats-new.txt

```diff
@@ -17,10 +17,15 @@ What's New
 .. meta::
    :keywords: bug fix, features
 
-Learn what's new by version:
+.. _kafka-connector-whats-new-1.13.1:
 
-* :ref:`Version 1.13 <kafka-connector-whats-new-1.13>`
-* :ref:`Version 1.12 <kafka-connector-whats-new-1.12>`
+What's New in 1.13.1
+--------------------
+
+- Fixed a bug where data copies itself again when a task restarts.
+  To learn more about this fix, see the 
+  `KAFKA-428 <https://jira.mongodb.org/browse/KAFKA-428>`__ JIRA issue.
+- Updates the ``ktlint`` version to 0.31.0.
```

Comment:
[nit] we could probably remove this from being highlighted on our page since it isn't too applicable externally.